src/main/java/teammates/common/util/Const.java

```diff
@@ -931,15 +887,13 @@ public static final String getjQueryUiFilePath(boolean isDevServer) {
         public static final String AUTOMATED_FEEDBACK_OPENING_REMINDERS = "/feedbackSessionOpeningReminders";
         public static final String AUTOMATED_FEEDBACK_CLOSING_REMINDERS = "/feedbackSessionClosingReminders";
         public static final String AUTOMATED_FEEDBACK_PUBLISHED_REMINDERS = "/feedbackSessionPublishedReminders";
-        public static final String AUTOMATED_COMPILE_LOGS = "/compileLogs";
         
         public static final String BACKDOOR = "/backdoor";
         
         //Task Queue Worker Servlets URI
         public static final String EMAIL_WORKER = "/emailWorker";
         public static final String ADMIN_EMAIL_WORKER = "/adminEmailWorker";
         public static final String ADMIN_EMAIL_PREPARE_TASK_QUEUE_WORKER = "/adminEmailPrepareTaskQueueWorker";
-        public static final String SUBMISSION_WORKER = "/submissionWorker";
```

Comment:
Pretty sure this is made obsolete when the `Evaluation`/`Submission` entities are migrated to `FeedbackSession`/`FeedbackQuestion`/`FeedbackResponse`.